src/test/resources/org/openmrs/uitestframework/test.properties

```diff
@@ -4,5 +4,5 @@ login.password=Admin123
 login.location=Pharmacy
 webdriver=firefox
 login.auto=false
-headless=false
-webdriver.gecko.driver=firefoxdriver/mac/geckodriver
\ No newline at end of file
+headless=true
```

Comment:
You are supposed not to change these file, Is this failing without these change.

Reply:
@sherrif10 I have already fixed the mistake I did. Thanks for the review though!